elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -473,8 +483,15 @@ export class PeerPool extends EventEmitter {
 		return this.getAllPeers().map(peer => peer.peerInfo);
 	}
 
-	public getAllPeers(): ReadonlyArray<Peer> {
-		return [...this._peerMap.values()];
+	public getAllPeers(
+		kind?: typeof OutboundPeer | typeof InboundPeer,
```

Comment:
function name should be adjusted here too because when we supply `kind` it's not `getAllPeers` anymore